scheduler/server/stateful_scheduler.go

```diff
@@ -267,12 +259,22 @@ func NewStatefulScheduler(
 	if config.MaxJobsPerRequestor == 0 {
 		config.MaxJobsPerRequestor = DefaultMaxJobsPerRequestor
 	}
-	if config.SoftMaxSchedulableTasks == 0 {
-		config.SoftMaxSchedulableTasks = DefaultSoftMaxSchedulableTasks
+	config.SchedAlgConfig = &LoadBasedAlgConfig{
+		rebalanceThreshold:   0,
+		rebalanceMinDuration: 0 * time.Minute,
```

Comment:
Zero values here are already both zero, so this is not needed.

Reply:
removed